rules_php_gapic/resources/readme.md

```diff
@@ -64,3 +64,13 @@ The recommended (tested) way of doing it is as follows:
     tar -zchpf php-7.4.15_linux_x86_64.tar.gz php-7.4.15/bin php-7.4.15/lib
     ```
 11. Copy the created archive to the `rules_gapic/php/resources` folder in this repository and post a PR.
+
+## Updating composer
+
+When generating client libraries, this generator is bundled together with a version of composer. Occasionally composer needs to be updated in order for bazel to build successfully. You may see a somewhat misleading error when trying to build with bazel with an out-of-date version of composer:
+
+```
+STDERR: Composer is operating significantly slower than normal because you do not have the PHP curl extension enabled. No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.
```

Comment:
I'm not sure that this error is correlated with an outdated version, so lets not mention it here. Furthermore, these docs are relevant even when there is nothing wrong, nut we just want to update composer.